client/src/components/loading/component.tsx

```diff
@@ -3,7 +3,7 @@ import type { LoadingProps } from './types';
 
 export const Loading: React.FC<LoadingProps> = ({ className }: LoadingProps) => (
   <svg
-    className={classNames('animate-spin -ml-1 mr-3 h-5 w-5 stroke-none', className)}
+    className={classNames('animate-spin h-5 w-5 stroke-none', className)}
```

Comment:
The reason for removing the margins from the generic component, is that it made it nigh impossible to set them on a per case basis.